### ISSUE TITLE
[Request]: Remove UI from image and option to remove the slideshow button

### ISSUE THREAD

**Issue by peterhoglund** [archetyped/simple-lightbox#855]
## Description of feature

Now the UI is always visible ontop of the image. I would like is either A) the UI fade away if the cursor is not over the image, or b) the UI be places outside of the image. Preferable A.

I would also like to be able to completely remove the slide show feature and not have the small playbutton in the UI at all.

Thanks!

## This feature would be used to

Be able to see the whole image without the UI cluttering it.

## If you added this feature, I would:

b) Jump for joy
c) Shave my cat

**Comment by BAProductions** [comment]
I agree, because Group items (for displaying as a slideshow) (Default: Enabled) remove the play pause button & the slideshow function, but also disable the next & previous button witch I also want to keep.
